lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -294,6 +294,14 @@ def add_creditcard(post, creditcard, options)
           end
 
           post[:card] = card
+
+          if creditcard.is_a?(NetworkTokenizationCreditCard)
```

Comment:
Could be a decrypted apple pay token, in general a credit card with the extra verification required for network tokenization